.cargo/config.toml

```diff
@@ -1,3 +0,0 @@
-[alias]
-# Temporary solution to have clippy config in a single place until https://github.com/rust-lang/rust-clippy/blob/master/doc/roadmap-2021.md#lintstoml-configuration is shipped.
-custom-clippy = "clippy --workspace --all-features --all-targets -- -A clippy::type_complexity -A clippy::pedantic -W clippy::used_underscore_binding -D warnings"
```

Comment:
Please don't remove this. We want to retain the alias to make it easy for users to run the command locally.

Reply:
Sure

Reply:
Done